projects/nx-verdaccio/src/plugin/targets/environment.targets.ts

```diff
@@ -114,13 +126,31 @@ export function getEnvTargets(
     },
     // runs env-bootstrap-env, install-env and stop-verdaccio
     [TARGET_ENVIRONMENT_SETUP]: {
+      // list of inputs that all subsequent tasks depend on
+      inputs: [
+        '{projectRoot}/project.json',
+        {
+          runtime: 'node --version',
+        },
+        {
+          runtime: 'npm --version',
+        },
+        {
+          externalDependencies: ['verdaccio'],
+        },
+        // depends on underlying project being e2e tested and its own dependencies
+        // ! it's important that implicitDependencies are correctly configured between this project and the project being tested
+        // '^build-artifacts',
+        '^production',
```

Comment:
Sweet!! Did not know this works. Thx, learned soemthing.

Reply:
regarding this comment:
>  it's important that implicitDependencies are correctly configured between this project and the project being tested

Should we do something about it? Checking that the e2e project has some `implicitDependencies` at least?

Reply:
Do you think we should try to e2e test it? And add docs if it is user configured?

I plan to lang the first e2e tests next week: #45

Reply:
for now we could just check that implicitDependencies are not empty in the project configuration (for envTargets/e2e projects only).

Reply:
> for now we could just check that implicitDependencies are not empty in the project configuration (for envTargets/e2e projects only).

Good idea!
Let's open a new issue for this. abd move the comment there.

Reply:
e2e tests green and merged! #45